pkgs/development/libraries/catboost/default.nix

```
@@ -0,0 +1,114 @@
+{ lib
+, config
+, stdenv
+, fetchFromGitHub
+, cmake
+, libiconv
+, llvmPackages
+, ninja
+, openssl
+, python3
+, ragel
+, yasm
+, zlib
+, cudaSupport ? config.cudaSupport or false
+, cudaPackages ? {}
+, enablePython ? false
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "catboost";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "catboost";
+    repo = "catboost";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-Ng1ylqL9fBRJP5hATv9FzJTqqSIFEUN0At1DJ3V2FM4=";
+  };
+
+  patches = [
+    ./remove-conan.patch
+    ./fix-imported-targets.patch
+  ] ++ lib.optionals (lib.versionOlder cudaPackages.cudatoolkit.version "11.8") [
+    ./remove-unsupport-cc.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace cmake/common.cmake \
+      --replace  "\''${RAGEL_BIN}" "${lib.getExe ragel}" \
+      --replace "\''${YASM_BIN}" "${lib.getExe yasm}"
+  '';
+
+  outputs = [
+    # binary and libraries e.g. bin/catboost, lib/catboost.so, etc.
+    "out"
+    # libraries required for Python, R, etc. support
+    # see https://catboost.ai/en/docs/installation/build-native-artifacts#targets
+    "dev"
+  ];
```

```suggestion
  outputs = [ "out" "dev" ];
```
this should follow the standard outputs outlines in the nixpkgs doc